src/data-visualisation/index.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: article
```

Comment:
Should be marked as `collection`.. even if its draft and not being shown.. we should use the intended layout.